### PR TITLE
Release 0.82.0

### PR DESCRIPTION
**Pull request by smacfarlane**
tenor-55641115

Signed-off-by: Scott Macfarlane <smacfarlane@chef.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,25 +2,29 @@
...
 #### New Features & Enhancements
 - Run health checks asynchronously [#6482](https://github.com/habitat-sh/habitat/pull/6482) ([christophermaier](https://github.com/christophermaier))
+- Be more liberal in our license acceptance checking. [#6574](https://github.com/habitat-sh/habitat/pull/6574) ([raskchanky](https://github.com/raskchanky))
 
 #### Bug Fixes
 - remove trailing space from env vars set in export docker build [#6540](https://github.com/habitat-sh/habitat/pull/6540) ([mwrock](https://github.com/mwrock))
 
 #### Merged Pull Requests
 - Move deadlock error to trace - stop spam [#6609](https://github.com/habitat-sh/habitat/pull/6609) ([chefsalim](https://github.com/chefsalim))
-- Update error handling for origin deletion [#6467](https://github.com/habitat-sh/habitat/pull/6467) ([chefsalim](https://github.com/chefsalim))
```
Comment (christophermaier) [inline CHANGELOG.md]: Why was this deleted?

Comment (smacfarlane) [reply CHANGELOG.md]: 🤦‍♂  Because I don't know how to drive vim.

Comment (smacfarlane) [reply CHANGELOG.md]: @christophermaier fixed

Comment (christophermaier) [reply CHANGELOG.md]: `vim` is a harsh master
